openid/openid.go

```diff
@@ -0,0 +1,242 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENSE file for details.
+
+// Package openid contains functions to help log-in to Ubuntu SSO using
+// OpenID 2.0.
+package openid
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/yohcop/openid-go"
+
+	"github.com/juju/usso"
+)
+
+const (
+	// These standard simple registration fields are supported by
+	// Ubuntu SSO.
+	SimpleRegistrationNickname = "nickname"
```

Comment:
"SimpleRegistration" seems like a big mouthful on every constant.
In particular, the "Simple" seems out of place and doesn't really
add any information.

How about just RegistrationNickname, RegistrationEmail, etc? Or even
just RegNickname etc?